modules/kms/main.tf

```diff
@@ -53,3 +53,11 @@ resource "google_kms_crypto_key" "default" {
     }
   }
 }
+
+resource "google_kms_key_ring_import_job" "default" {
+  count            = var.import_job_create ? 1 : 0
```

Comment:
Looks like you can get away with a single variable. The condition here can be `var.import_job != null ? 1 : 0`

Reply:
Hi @juliocc I don't know why I went with an additional variable :/ I will add this change.